.github/workflows/tests-php.yml

```diff
@@ -137,9 +138,16 @@ jobs:
         run: |
           ${SLIC_BIN} use the-events-calendar
           ${SLIC_BIN} composer install
+      # ------------------------------------------------------------------------------
+      # Update WordPress version to our min required.
+      # ------------------------------------------------------------------------------
       - name: Init the WordPress container
         if: steps.skip.outputs.value != 1
-        run: ${SLIC_BIN} up wordpress
+        run: |
+          ${SLIC_BIN} up wordpress
+          ${SLIC_BIN} wp core version
+          ${SLIC_BIN} wp core update --force --version=6.3.1
```

Comment:
WP 6.3.1 is too old, the other plugins are using `6.4.1` so at least we should try to match that.